lib/plugins/aws/package/compile/events/apiGateway/lib/method/authorization.js

```diff
@@ -5,7 +5,7 @@ const awsArnRegExs = require('../../../../../../utils/arnRegularExpressions');
 
 module.exports = {
   getMethodAuthorization(http) {
-    if (_.get(http, 'authorizer.type') === 'AWS_IAM') {
+      if (_.get(http, 'authorizer.type') === 'AWS_IAM') {
```

Comment:
Are you sure it's safe (is `authorizer` guaranteed` to be set on `http` object?

Reply:
In addition, looking at aws documentation, its optional for users to set an authorizer.

Reply:
AWS documentation is irrelevant here. We're processing Framework config